service/src/main/java/org/mskcc/cmo/metadb/service/impl/SampleServiceImpl.java

```diff
@@ -133,7 +133,10 @@ public Boolean sampleHasMetadataUpdates(SampleMetadata existingSampleMetadata,
     @Override
     public PublishedMetadbSample getPublishedMetadbSample(UUID metadbSampleId) throws Exception {
         MetadbSample sample = getMetadbSample(metadbSampleId);
-        return new PublishedMetadbSample(sample);
+        PublishedMetadbSample publishedSample = new PublishedMetadbSample(sample);
+        publishedSample.setIsCmoSample(Boolean.valueOf(sampleRepository
```

Comment:
This is where i add `isCmoSample` field. open to suggestions for a better name than isCmoSample

Reply:
[requested change] throw this into `additionalProperties` map

Reply:
also can remove call to query below

Reply:
reminder: remove from published sample and expand published sample with `additionalProperties` instead, which is stored in the `SampleMetadata` node already 

then add `isCmo` to the additional properties for the published sample returned